python/ic_azure/azure_discovery.py

```diff
@@ -8,25 +8,30 @@
 from azure_client import AzureClient
 
 
-class InsightsDiscovery(object):
-    """Discover available metrics from Azure's Insights components."""
+class AzureDiscovery(object):
+    """Discover available metrics from Azure's components."""
 
     def __init__(self, azure_client):
         self._client = azure_client.client()
         self.subscription_id = azure_client.subscription_id
 
-    def find_services(self, resource_group, resource_name):
+    def find_services(self, resource_group, provider_name, resource_type,
+                      resource_name):
         servicesList = []
 
         # Create resource ID
         resource_id = "subscriptions/{}/resourceGroups/{}".format(
             self.subscription_id,
             resource_group
         )
-        resource_id += "/providers/microsoft.insights/components/{}".format(
+        resource_id += "/providers/{}/{}/{}".format(
+            provider_name,
+            resource_type,
             resource_name
         )
 
+        print resource_id
```

Comment:
This should not be printed in discovery output.

Reply:
Nice catch! This was for debugging purposes and was removed in commit 09df7ddb0e7d03dd24204998bab922a24ca136de.